examples/bootstrap/bootstrap.sh

```diff
@@ -42,7 +42,9 @@ $MANAGER > $LOGDIR/manager.log 2>&1 --port=8080 --expanderURL=http://localhost:8
 echo
 
 echo "Creating dm namespace..."
-$KUBECTL create -f dm-namespace.yaml
+BOOTSTRAP_PATH=$( cd $(dirname $0) ; pwd -P )
+$KUBECTL delete -f $BOOTSTRAP_PATH/dm-namespace.yaml
```

Comment:
Ack on adding the path.

Is adding the delete any better?  Now instead of create failing when it already exists, delete fails when it doesn't exist.  Another option is to do 'kubectl get namespaces dm', and switch on the $? result to create only when it doesn't exist.